src/Program.spec.ts

```diff
@@ -662,9 +664,9 @@ describe('Program', () => {
         it('picks up new files in a scope when an xml file is loaded', async () => {
             program.options.ignoreErrorCodes.push(1013);
             let xmlPath = s`${rootDir}/components/component1.xml`;
-            await program.addOrReplaceFile({ src: xmlPath, dest: 'components/comonent1.xml' }, `
+            await program.addOrReplaceFile({ src: xmlPath, dest: 'components/comonent1.xml' }, trim`
                 <?xml version="1.0" encoding="utf-8" ?>
-                <component name="HeroScene" extends="Scene" >');
```

Comment:
Interesting that these tests still succeeded with this junk in the XML.

Reply:
I know right :D